src/vs/workbench/contrib/notebook/browser/contrib/find/notebookFindReplaceWidget.ts

```diff
@@ -173,14 +173,6 @@ export class NotebookFindInputFilterButton extends Disposable {
 		this._filterButtonContainer = dom.$('.find-filter-button');
 		this._filterButtonContainer.classList.add('monaco-custom-toggle');
 		this.createFilters(this._filterButtonContainer);
-
-		this._register(this.filters.onDidChange(() => {
-			if (this.filters.codeInput !== true || this.filters.codeOutput !== true || this.filters.markupInput !== true || this.filters.markupPreview !== false) {
-				this._filtersAction.checked = true;
-			} else {
-				this._filtersAction.checked = false;
-			}
-		}));
```

Comment:
did you mean to remove this?

Reply:
yes, I found this unnecessary as it's handled outside in NotebookFindReplaceWidget already. Is it being used by Search?

Reply:
Right, it's handled outside also. That's fine then!